src/com/duplicatesSample/DuplicatesSample.java

```diff
@@ -0,0 +1,42 @@
+package com.duplicatesSample;
+
+public class DuplicatesSample {
+
+	public static void main(String args[]) {
+		 char[] array1=new char[10];
+	     char[] array2=new char[10];
+	         array1[0]='c';
+	         array1[1]='a';
+	         array1[2]='b';
+	         array1[3]='c';
+	     array1[4]='a';
+	     array1[5]='e';
+	     array1[6]='e';
+	     array1[7]='a';
+	     array1[8]='e';
+	     array1[9]='d';
+	     for(int i=0;i<10;i++)
```

Comment:
use "array1.length" instead of '10' as hardcoded value
```suggestion
	     for(int i=0;i<10;i++)
```